common/dm.c

```diff
@@ -77,6 +78,27 @@ dm_get_by_name(const char *name)
 	panic("dm: Device %s not found", name);
 }
 
+int
+dm_set_pins(struct device *dev, uint8_t num_pins)
+{
+	struct gpio_pin *pins = dev->gpio_pins;
+	struct device   *gpio_dev;
+	uint8_t pin_id, mode;
+	int     result;
+
+	for (uint8_t p = 0; p < num_pins; ++p) {
```

Comment:
Please make this `size_t i` to match everywhere else

Reply:
Done.